app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/add/FetchTestRecordFragment.kt

```diff
@@ -94,6 +92,20 @@ class FetchTestRecordFragment : Fragment(R.layout.fragment_fetch_covid_test_resu
         }
     }
 
+    private fun handleError(state: FetchTestRecordUiState) {
```

Comment:
This condition should be checked on ViewModel instead of the fragment.
ViewModel should pass the error object with title & message.

Reply:
done